docs/src/labels_annotations.md

```diff
@@ -77,6 +77,10 @@ These predefined labels are managed by CloudNativePG.
 : Available on `ConfigMap` and `Secret` resources. When set to `true`,
   a change in the resource is automatically reloaded by the operator.
 
+`cnpg.io/user-role`
```

Comment:
`user` and `role`  are quasi-synonyms in Postgres.
How about `cnpg.io/user-class` or similar?